target_postgres/postgres.py

```diff
@@ -622,25 +622,20 @@ def get_postgres_datetime(self, *args):
     def add_column(self, cur, table_schema, table_name, column_name, column_schema):
         data_type = json_schema.to_sql(column_schema)
 
-        default_value = self.get_null_default(column_name, column_schema)
-
-        if default_value is not None:
-            default_value = sql.SQL(' DEFAULT {}').format(sql.Literal(default_value))
-        elif not self.is_table_empty(cur, table_schema, table_name):
-            raise PostgresError('Non-trival default needed on new non-null column `{}.{}.{}`'.format(
+        if not json_schema.is_nullable(column_schema) \
```

Comment:
@AlexanderMann Aren't we always forcing columns to be not nullable, regardless if the table is empty?